src/instance/globus-groups.json

```diff
@@ -0,0 +1,393 @@
+[
```

Comment:
Since the globus_groups support is removed, can we also remove this HuBMAP groups json file for now to avoid confusion?

Reply:
Yes will do